spec/features/validation_errors/list_validation_errors_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "List validation errors" do
+  context "as a system admin" do
+    let(:user) { create(:user, system_admin: true) }
```

Comment:
We're trying to avoid `let` blocks in feature files where possible (I know other files have them, but as a team we agreed to keep the rspec sugar to a minimum and use "steps" to drive out the state, journey and expectations as much as possible). Could we change it to something like this?

```ruby
scenario "list validation_errors" do
  given_i_am_authenticated_as_system_admin
  and_a_validation_error_exists
  when_i_visit_the_validation_errors_index_page
  then_i_see_the_validation_errors
end

def and_a_validation_error_exists
  @validation_error = create(:validation_error)
end
```

In `spec/support/features/authentication_steps.rb` we can have a reusable step:

```ruby
def given_i_am_authenticated_as_system_admin
  given_i_am_authenticated(user: create(:user, :system_admin))
end
```